server.js

```diff
@@ -21,45 +22,36 @@ app.use(express.static('public'));
 // body parser config to accept our datatypes
 app.use(bodyParser.urlencoded({ extended: true }));
 
+// allows prgm to process our schema/objects
+var Book = require('./models/book');
```

Comment:
You don't need this variable since you're requiring the entire models folder on line 35.